recipe/cache.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Utility\Httpie;
+
+desc('Reset the opcache using a file strategy');
+task('sumo:opcache:reset-file', function () {
+    $opcacheResetScript = 'opcache_reset.php';
```

Comment:
Maybe you should align it with https://github.com/sumocoders/sumodev_deploy/blob/master/lib/sumodev_deploy/tasks/opcache.rb#L10

I think, the php-opcache-reset.php file is allowed in .htaccess in Fork to be accessed directly.

Reply:
I changed the name to comply with the filename used in Fork. But I still have a problem, because the path to the file is different in Fork. In Symfony the file exists in the `public` directory but in Fork the exists in the root of the project, and I don't know exactly how to specify the path according to the platform being ran. Any ideas @tijsverkoyen ?

Reply:
Why not use the current path? As this is done after the symlink has happened.
`{{deploy_path}}/current')`

Reply:
The path can be built with `document_root` and this way it works fine.